cockatrice/src/releasechannel.cpp

```diff
@@ -290,20 +304,21 @@ void DevReleaseChannel::fileListFinished()
     foreach(QVariant file, resultList)
     {
         QVariantMap map = file.toMap();
-        if(!map.contains("version"))
-            continue;
-        if(!map["version"].toString().endsWith(shortHash))
+
+        // File name _MUST_ contain the build hash
```

Comment:
Quite awkward. Do we have to have it this way? Can't we just go off the tag name + file?

Dev channel updater should release stable releases if that's the latest upload